kubectl argo rollouts create analysisrun segfaults

I ran the following:

```
kubectl-argo-rollouts create analysisrun --from post-promotion -n xxxxx
```

and got:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x23b6dc3]

goroutine 1 [running]:
github.com/argoproj/argo-rollouts/pkg/kubectl-argo-rollouts/cmd/create.(*CreateAnalysisRunOptions).getAnalysisTemplate(0xc000248240, 0x0, 0x0, 0x0)
	/go/src/github.com/argoproj/argo-rollouts/pkg/kubectl-argo-rollouts/cmd/create/create.go:328 +0x1a3
github.com/argoproj/argo-rollouts/pkg/kubectl-argo-rollouts/cmd/create.NewCmdCreateAnalysisRun.func1(0xc00069c840, 0xc0001a7580, 0x0, 0x4, 0x0, 0x0)
	/go/src/github.com/argoproj/argo-rollouts/pkg/kubectl-argo-rollouts/cmd/create/create.go:270 +0x7c5
github.com/spf13/cobra.(*Command).execute(0xc00069c840, 0xc0001a7540, 0x4, 0x4, 0xc00069c840, 0xc0001a7540)
	/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:850 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xc00069c2c0, 0xc00069c2c0, 0x3278760, 0xc0001a0008)
	/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
main.main()
	/go/src/github.com/argoproj/argo-rollouts/cmd/kubectl-argo-rollouts/main.go:21 +0xc5
```

Kubectl version:

```
Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.0", GitCommit:"c2b5237ccd9c0f1d600d3072634ca66cefdf272f", GitTreeState:"clean", BuildDate:"2021-08-04T17:56:19Z", GoVersion:"go1.16.6", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"19+", GitVersion:"v1.19.13-eks-8df270", GitCommit:"8df2700a72a2598fa3a67c05126fa158fd839620", GitTreeState:"clean", BuildDate:"2021-07-31T01:36:57Z", GoVersion:"go1.15.14", Compiler:"gc", Platform:"linux/amd64"}
WARNING: version difference between client (1.22) and server (1.19) exceeds the supported minor version skew of +/-1
```

Kubectl argo rollouts version:

```
kubectl-argo-rollouts: v1.0.4+c63b6c1
  BuildDate: 2021-08-03T02:45:19Z
  GitCommit: c63b6c1f5134a9f19caae37765f1a8a145f62d7d
  GitTreeState: clean
  GoVersion: go1.16.3
  Compiler: gc
  Platform: darwin/amd64
```

This issue happens for all versions of kubectl argo rollouts higher than 0.9.1.

--- a/pkg/kubectl-argo-rollouts/cmd/create/create.go
+++ b/pkg/kubectl-argo-rollouts/cmd/create/create.go
@@ -244,6 +244,7 @@ func NewCmdCreateAnalysisRun(o *options.ArgoRolloutsOptions) *cobra.Command {
 		SilenceUsage: true,
 		RunE: func(c *cobra.Command, args []string) error {
 			ctx := c.Context()
+			createOptions.DynamicClientset()
 			froms := 0
 			if createOptions.From != "" {
 				froms++

